modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json

```
@@ -1462,51 +1326,213 @@
-																					"id": "element-text",
-																					"value": {
-																						"fragmentLink": {
-																						},
-																						"text": {
-																							"mapping": {
-																								"fieldKey": "ddmTemplate__ddmTemplate_8811568",
-																								"itemReference": {
-																									"contextSource": "DisplayPageItem"
-																								}
-																							}
-																						}
+																					"id": "landscapeMobile",
+																					"rowViewportDefinition": {
+																						"modulesPerRow": 1
+																					}
+																				},
+																				{
+																					"id": "portraitMobile",
+																					"rowViewportDefinition": {
+																					}
+																				},
+																				{
+																					"id": "tablet",
+																					"rowViewportDefinition": {
 																					}
 																				}
 																			],
-																			"indexed": true
+																			"verticalAlignment": "middle"
 																		},
-																		"type": "Fragment"
+																		"pageElements": [
+																			{
+																				"definition": {
+																					"columnViewports": [
+																						{
+																							"columnViewportDefinition": {
+																								"size": 12
+																							},
+																							"id": "landscapeMobile"
+																						},
+																						{
+																							"columnViewportDefinition": {
+																							},
+																							"id": "portraitMobile"
+																						},
+																						{
+																							"columnViewportDefinition": {
+																							},
+																							"id": "tablet"
+																						}
+																					],
+																					"size": 12
+																				},
+																				"pageElements": [
+																					{
+																						"definition": {
+																							"indexed": true,
+																							"layout": {
+																								"align": "Center",
+																								"borderWidth": 0,
+																								"contentDisplay": "FlexRow",
+																								"marginBottom": 0,
+																								"marginLeft": 0,
+																								"marginRight": 0,
+																								"marginTop": 0,
+																								"opacity": 100,
+																								"paddingBottom": 0,
+																								"paddingLeft": 0,
+																								"paddingRight": 0,
+																								"paddingTop": 0,
+																								"widthType": "Fluid"
+																							}
+																						},
+																						"pageElements": [
+																							{
+																								"definition": {
+																									"fragment": {
+																										"key": "icon",
+																										"siteKey": "Marketplace"
```

Please make the replacement
```suggestion
																										"siteKey": "[$GROUP_KEY$]"
```